tool/ci.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
-# Created with package:mono_repo v6.6.2
+# Created with package:mono_repo v6.5.1
```

Comment:
You may need to update your local mono_repo version (`dart pub global activate mono_repo`).